### PR TITLE
FSET-1767 Added email template for Sift ready progress status

### PR DESCRIPTION
**Pull request by WarFox**
Corresponding backend branch here is WIP
https://github.com/hmrc/fset-faststream/tree/fset-1767

### CODE REVIEW

**Comment by WarFox** [comment]
Back-end pull request here https://github.com/hmrc/fset-faststream/pull/503

File: app/uk/gov/hmrc/fsetemailrenderer/templates/faststream/fsetFaststreamNotifyCandidateSiftEnteredAdditionalQuestions.scala.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    <h3 style="font-size: 32px; line-height: 1.315789474; margin: 0 0 30px 0;">At least one of your scheme preferences require you to answer some additional questions.</h3>
+
+    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">You should complete this within 7 days from the date of your email.
+        Failure to do this will hold up your application.</p>
+
+    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">Sign in to continue <br>(<a href="/faststream/profile-signin.html?Status=attend">https://www.apply-civil-service-fast-stream.service.gov.uk/signin</a>)</p>
+
+    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">If you no longer wish to continue, please sign in and withdraw your application. We'll then stop sending any further emails.</p>
+
+    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">Your password reset code will expire after an hour.</p>
```
Comment (henricook) [inline app/uk/gov/hmrc/fsetemailrenderer/templates/faststream/fsetFaststreamNotifyCandidateSiftEnteredAdditionalQuestions.scala.html]: @WarFox this mentions a password reset code, should this be here?

Comment (WarFox) [reply app/uk/gov/hmrc/fsetemailrenderer/templates/faststream/fsetFaststreamNotifyCandidateSiftEnteredAdditionalQuestions.scala.html]: great find @henricook! Also, the sign-in URL must be updated

File: app/uk/gov/hmrc/fsetemailrenderer/preview/TemplateParams.scala [hunk 2]
```diff
@@ -24,8 +26,31 @@ object TemplateParams extends TemplateParams {
...
+    "assessmentCenterAdjustments" -> "Assessment center adjustments",
+    "scheme" -> "Scheme"
+  )
+
   val sampleTemplateParameters: Map[String, Params] = Map(
     "sample1" -> Params(Map("name" -> "Mr. Hyde")),
     "sample2" -> Params(Map("name" -> "Dr. Jekyll"))
-  )
+  ).withDefaultValue(Params(defaultParameters))
+
```
Comment (henricook) [inline app/uk/gov/hmrc/fsetemailrenderer/preview/TemplateParams.scala]: Can you tell me about the impact of this change?

Comment (WarFox) [reply app/uk/gov/hmrc/fsetemailrenderer/preview/TemplateParams.scala]: This fixes preview feature with default values for many template variables. You can also override them with query parameters

File: app/uk/gov/hmrc/fsetemailrenderer/preview/TemplateParams.scala [hunk 3]
```diff
@@ -24,8 +26,31 @@ object TemplateParams extends TemplateParams {
 trait TemplateParams {
   def paramsFor(templateId: String): Params = sampleTemplateParameters(templateId)
 
+  val defaultParameters = Map(
```
Comment (henricook) [inline app/uk/gov/hmrc/fsetemailrenderer/preview/TemplateParams.scala]: Please make this private/protected if it's only used inside the class

**Comment by henricook** [comment]
Put in the private/protected for me then good to merge

**Pull request merged**